third_party/3/orjson.pyi

```diff
@@ -0,0 +1,14 @@
+# https://github.com/ijl/orjson/blob/master/orjson.pyi
+
+from typing import Any, Callable, Optional, Union
+
+def dumps(
+    obj: Any, default: Optional[Callable[[Any], Any]], option: Optional[int]
+) -> bytes: ...
+def loads(obj: Union[bytes, str]) -> Union[dict, list, int, float, str, None]: ...
```

Comment:
It's probably more convenient for users to just return `Any` here, until mypy supports recursive types and we can define a proper JSON type. The stubs for `json` do the same thing:

```python
def loads(s: _LoadsString,
          encoding: Any = ...,  # ignored and deprecated
          cls: Any = ...,
          object_hook: Optional[Callable[[Dict], Any]] = ...,
          parse_float: Optional[Callable[[str], Any]] = ...,
          parse_int: Optional[Callable[[str], Any]] = ...,
          parse_constant: Optional[Callable[[str], Any]] = ...,
          object_pairs_hook: Optional[Callable[[List[Tuple[Any, Any]]], Any]] = ...,
          **kwds: Any) -> Any: ...
```

Otherwise users are forced to use casts, isinstance checks or `# type: ignore` when dealing with dicts and lists.

Reply:
Ok.